docs/Conditional-processing-and-comment-syntax.md

```diff
@@ -352,14 +366,14 @@ In this sample, we can see that if the **TargetFrameworkOverride** symbol is def
 </ItemGroup>
 ```
 
+In this sample, it can also use conditional expressions inside an xml type comment for MSBuild files.
```

Comment:
`In this snippet of MSBuild file, we see usage of conditional expression embeded inside a xml comment.`